industrial_robot_client/src/joint_trajectory_action.cpp

```diff
@@ -224,7 +226,7 @@ void JointTrajectoryAction::controllerStateCB(const control_msgs::FollowJointTra
     return;
   }
 
-  if (ros::Time::now() < time_to_check_)
+  if (!has_moved_once_ && (ros::Time::now() < time_to_check_))
```

Comment:
`has_moved_once_` is only set if the robot status message is also published.  Not all robots publish this message, see [here](https://github.com/simonschmeisser/industrial_core/blob/indigo-devel/industrial_robot_client/src/joint_trajectory_action.cpp#L268).  I believe an additional check to `last_robot_status_`, like [here](https://github.com/simonschmeisser/industrial_core/blob/indigo-devel/industrial_robot_client/src/joint_trajectory_action.cpp#L241) would make this change compatible with older drivers.